docs/api/browser-window.md

```diff
@@ -219,9 +219,14 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       bar.
     * `hidden` - Results in a hidden title bar and a full size content window, yet
       the title bar still has the standard window controls ("traffic lights") in
-      the top left.
+      the top left.  Using this value will also enable the Window Controls Overlay
+      [JavaScript APIs][overlay-javascript-apis] and
```

Comment:
Can you mention in documentation that `WebAppWindowControlsOverlay` must be explicitly enabled in blink features to make the APIs work?